templates/pyodide-console.html

```diff
@@ -0,0 +1,267 @@
+<!doctype html>
```

Comment:
Can we find a new home for this file? Maybe under `test/` for Emscripten?

Did you write this file yourself or is it from somewhere, I don't see license information.

What is the expected maintenance requirements for this file?

Reply:
> Can we find a new home for this file? Maybe under test/ for Emscripten?

Sure, I can do that. 


> Did you write this file yourself or is it from somewhere, I don't see license information.

I mentioned in the PR description 
>> creates a index.html file in dist/, with a pyodine terminal taken
from [pyodide src/templates/console.html]

The LICENSE is https://github.com/pyodide/pyodide/blob/main/LICENSE. (Mozilla Public License Version 2.0) 

I can add a comment on the top of the file with the original source url, not sure , should I add a "copyright" comment although the original does not have it? The closest copyright notice that I can find in their repo is  "2019-2022, Pyodide contributors and Mozilla", I can add that?  


> What is the expected maintenance requirements for this file?

I guess we won't maintain this file. It's the same jQuery REPL that is used in https://pyodide.org/en/stable/console.html, if it stops working we can always copy the latest from [pyodide src/templates/console.html]. And then add the lines

```
        await term.ready;
        await term.exec("import micropip\n");
        await term.exec("micropip.list()\n");
        await term.exec('await micropip.install("http://localhost:8000/urllib3-2.2.0-py3-none-any.whl")')
        await term.exec("micropip.list()");
        await term.exec("import urllib3");
        await term.exec("urllib3.__version__");
```

 [pyodide src/templates/console.html]: https://github.com/pyodide/pyodide/blob/main/src/templates/console.html

Reply:
Thank you, apologies for not reading the description closely enough